en/00_Getting_Started/02_Composer.md

```diff
@@ -74,21 +74,21 @@ You can find thousands of Silverstripe CMS modules on [Packagist](https://packag
 Installing a module can be done with [the require command](https://getcomposer.org/doc/03-cli.md#require-r):
 
 ```bash
-composer require silverstripe/blog
+composer require dnadesign/silverstripe-elemental
```

Comment:
Unrelated but since blog isn't commercially supported I'm removing the reference.